src/routes/admins.js

```diff
@@ -72,13 +68,14 @@ export default (apiServer) => {
     }
   })
 
-  apiServer.patch('/v1/admins/:id/password', async req => {
+  apiServer.patch('/v1/admins/:id/password', async req => { /// ///////////////////////Not updating the password
     allowAccessTo(req, secrets, [{ type: 'admin', user: { _id: req.params.id } }]) // check auth
     if (req.body.newPassword !== req.body.newPasswordAgain) { // check password matching
       throw new ValidationError("Validation error passwords didn't match ")
     }
     const hash = crypto.createHash('md5').update(req.body.newPassword).digest('hex') // hash the new password
-    await patchOne(AdminModel, { id: req.params.id }, { password: hash }) // update user password
+    const oldHash = crypto.createHash('md5').update(req.body.oldPassword).digest('hex') // hash the new password
+    await patchOne(AdminModel, { id: req.params.id, password: oldHash }, { password: hash }) // update user password
```

Comment:
I think you don't need the old hash here, because then we would need a mongodb index that supports this kind of query. I believe it's better to get the item from the db, check if the hashes are the same and then update with the new hash.